### ISSUE TITLE
VS Code Extension Logic Apps (Preview) does not work with .NET 5

### ISSUE THREAD

**Issue by PalmEmanuel** [Azure/logicapps#160]
When I try to create a Logic App using the new VS Code extension Logic Apps (Preview) v0.0.1 I get an error that may be the same described in the issue [#2153](https://github.com/microsoft/vscode-azurefunctions/issues/2153).

"You must have the [.NET Core SDK](https://aka.ms/AA4ac70) installed to perform this operation. See [here](https://aka.ms/AA1tpij) for supported versions."

```plaintext
PS F:\Code\LocalLogicApp> dotnet --list-sdks
2.1.202 [C:\Program Files\dotnet\sdk]
2.1.402 [C:\Program Files\dotnet\sdk]
2.1.403 [C:\Program Files\dotnet\sdk]
2.1.511 [C:\Program Files\dotnet\sdk]
2.1.700 [C:\Program Files\dotnet\sdk]
2.1.801 [C:\Program Files\dotnet\sdk]
2.2.300 [C:\Program Files\dotnet\sdk]
3.1.100 [C:\Program Files\dotnet\sdk]
3.1.402 [C:\Program Files\dotnet\sdk]
5.0.100-rc.1.20452.10 [C:\Program Files\dotnet\sdk]
```

When I uninstall .NET 5 it works.

**Comment by hajekj** [comment]
What worked for me was to create a `global.json` referencing 3.X runtime instead of uninstalling .NET 5.

This has been fixed here: https://github.com/microsoft/vscode-azurefunctions/issues/2153 probably didn't make it into the preview release.

**Comment by PalmEmanuel** [comment]
> What worked for me was to create a `global.json` referencing 3.X runtime instead of uninstalling .NET 5.
> 
> This has been fixed here: [microsoft/vscode-azurefunctions#2153](https://github.com/microsoft/vscode-azurefunctions/issues/2153) probably didn't make it into the preview release.

That's a reasonable workaround, thanks! I figured it should be recorded here as well since I didn't find an open issue but if a fix is already in the scope for a future release this issue can be closed.

**Comment by joechung-msft** [comment]
0.0.5 release of the extension has a fix to resolve this.
